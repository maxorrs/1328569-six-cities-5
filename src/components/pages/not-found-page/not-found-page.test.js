import React from 'react';
import {BrowserRouter} from 'react-router-dom';
import renderer from 'react-test-renderer';

import NotFoundPage from './not-found-page';

it(`NotFoundPage is rendered correctly`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <NotFoundPage />
        </BrowserRouter>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
