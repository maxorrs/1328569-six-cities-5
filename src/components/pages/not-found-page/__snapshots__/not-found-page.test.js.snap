// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage is rendered correctly 1`] = `
<div
  className="not-found"
>
  <h1
    className="not-found__title"
  >
    <span
      className="not-found__error-number"
    >
      404.
    </span>
    <span
      className="not-found__error-description"
    >
      Page not found
    </span>
  </h1>
  <a
    className="not-found__link"
    href="/"
    onClick={[Function]}
  >
    Go to main page
  </a>
</div>
`;
